jenkins.go

```diff
@@ -503,7 +512,7 @@ func CreateJenkins(base string, auth ...interface{}) *Jenkins {
 		base = base[:len(base)-1]
 	}
 	j.Server = base
-	j.Requester = &Requester{Base: base, SslVerify: false}
+	j.Requester = &Requester{Base: base, SslVerify: true}
```

Comment:
j.Requester.CACert is not initialized, will be set anywhere else? Not sure whether CACert is needed, as if tls.Config.RootCAs is nil, TLS uses the host's root CA set.